src/joint.dia.graph.js

```diff
@@ -51,6 +51,22 @@ joint.dia.GraphCells = Backbone.Collection.extend({
     comparator: function(model) {
 
         return model.get('z') || 0;
+    },
+
+    startBatch: function(name, opt) {
+        return this.trigger('batch:start', _.extend({}, opt, { element: this, batchName: name }));
```

Comment:
Note that `this` context is not a `joint.dia.Element` but `joint.dia.Graph` here.
Batches in general doesn't have to be related to a single element. It could be related to a set of elements and links (e.g subgraph removal).
I suggest not to mention the `element` attribute here at all.